### ISSUE TITLE
Add Gitlab

### ISSUE THREAD

**Issue by shubheksha** [freeCodeCamp/how-to-contribute-to-open-source#6]
[Dev Kit](https://gitlab.com/gitlab-org/gitlab-development-kit)
[Dev Guide](https://docs.gitlab.com/ce/development/README.html)
Issue label: [up-for-grabs](https://gitlab.com/gitlab-org/gitlab-ce/issues?label_name=up-for-grabs)

**Comment by raj-maurya** [comment]
@shubheksha 

Can you please, describe a little more about this issue, what is expected to do?

**Comment by shovanch** [comment]
Hello, Can I take this issue? Would love to contribute to open source first time.

**Comment by CristyTarantino** [comment]
Is this issue still relevant? Would it be possible to have some more details?

**Comment by erictleung** [comment]
@CristyTarantino yes, this can still be worked on!

The file you want to edit is [this one](https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/master/PROJECTS.md) and the "code" you'll want to add is:

```
- [GitLab](https://gitlab.com/gitlab-org) (org)
    - [Contributing Guide](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md)
    - [Development Guides](https://docs.gitlab.com/ce/development/README.html)
    - [Development Kit](https://gitlab.com/gitlab-org/gitlab-development-kit)
    - Issue labels:
        - [1st contribution](https://gitlab.com/gitlab-org/gitlab-ce/issues?scope=all&utf8=%E2%9C%93&state=opened&label_name[]=1st%20contribution)
```

**Comment by CristyTarantino** [comment]
@erictleung I will make a pull request then. Thank you so much for the opportunity.

**Comment by shubheksha** [comment]
Closed in #217
